taccsite_cms/static/site_cms/js/modules/setTargetForExternalLinks.js

```diff
@@ -12,57 +12,22 @@ const SHOULD_DEBUG = window.DEBUG;
 
```

Comment:
@wesleyboar Should I keep lines 7-11 as well?

Reply:
Yes please.

Reply:
Sounds good. Resolving.